spec/acceptance/compose_v3_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper_acceptance'
+require 'pry'
```

Comment:
this is probably not required

Reply:
@florindragos good spot Florin. Ill get that removed now.